src/satosa/micro_services/ldap_attribute_store.py

```diff
@@ -19,88 +20,139 @@ class LdapAttributeStore(satosa.micro_services.base.ResponseMicroService):
     to lookup a person record in LDAP and obtain attributes
     to assert about the user to the frontend receiving service.
     """
+    logprefix = "LDAP_ATTRIBUTE_STORE:"
 
     def __init__(self, config, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.config = config
 
     def process(self, context, data):
+        logprefix = LdapAttributeStore.logprefix
+
+        # Initialize the configuration to use as the default configuration
+        # that is passed during initialization.
+        config = self.config
+        configClean = copy.deepcopy(config)
+        if 'bind_password' in configClean:
+            configClean['bind_password'] = 'XXXXXXXX'    
```

Comment:
No. The intent is to log the configuration at the DEBUG level but blank out any bind password included in the configuration so that passwords are never logged.